examples/rcn/load_data.py

```diff
@@ -0,0 +1,78 @@
+# export
+import os
+
+import cv2
+import matplotlib.pyplot as plt
+import numpy as np
+
+
+# export
+def get_mnist_data_iters(data_dir, train_size, test_size, full_test_set=False, seed=5):
+    """
+    Load MNIST data.
+
+    Assumed data directory structure:
```

Comment:
Not sure we want to assume this kind of data directory structure....Better to work with a loaded dataset where things are already organized into arrays.